controllers/comments.controller.js

```diff
@@ -0,0 +1,17 @@
+const { fetchCommentsByArticleId } = require("../models/comments.model");
+const { fetchArticleById } = require("../models/articles.model");
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  Promise.all([
+    fetchCommentsByArticleId(article_id),
+    fetchArticleById(article_id),
+  ])
+    .then(([comments, article]) => {
+      if (!article) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
```

Comment:
I think you could probably get rid of this and it would still work. I think if there isnt an article, then it will have rejected, so wont actually get here. 

might be wrong, but i'd give it a go and see if it still passes the test